vowpalwabbit/rand48.cc

```diff
@@ -20,6 +20,14 @@ float merand48(uint64_t& initial) {
   return bits_to_float(temp) - 1;
 }
 
+#ifdef __APPLE__
```

Comment:
Without this VW doesn't build on my machine.  Here is the output

```
/usr/bin/g++ -std=c++0x   -D__extern_always_inline=inline -Wall -pedantic -DNDEBUG -O3 -fomit-frame-pointer -fno-strict-aliasing  -D_FILE_OFFSET_BITS=64 -I /opt/local/include  -fPIC  -c oaa.cc -o oaa.o
parse_regressor.cc:40:24: error: no matching function for call to 'merand48'
{*iter = (float)(0.1 * merand48(ind));
                       ^~~~~~~~
./rand48.h:5:7: note: candidate function not viable: no known conversion from 'size_t' (aka 'unsigned long') to 'uint64_t &' (aka 'unsigned long long &') for 1st argument
float merand48(uint64_t& initial);
      ^
parse_regressor.cc:44:18: error: no matching function for call to 'merand48'
{*iter = (float)(merand48(ind) - 0.5);
                 ^~~~~~~~
./rand48.h:5:7: note: candidate function not viable: no known conversion from 'size_t' (aka 'unsigned long') to 'uint64_t &' (aka 'unsigned long long &') for 1st argument
float merand48(uint64_t& initial);
      ^
2 errors generated.
make[1]: *** [parse_regressor.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [vw] Error 2
```

I found the fix [here](http://stackoverflow.com/questions/11603818/why-is-there-ambiguity-between-uint32-t-and-uint64-t-when-using-size-t-on-mac-os).